tuf-spec.md

```diff
@@ -688,9 +688,11 @@ The "signed" portion of <a>root.json</a> is as follows:
 
   : <dfn>CONSISTENT_SNAPSHOT</dfn>
   ::
-    A boolean indicating whether the repository supports
-    consistent snapshots.  Section 7 goes into more detail on the consequences
-    of enabling this setting on a repository.
+    An optional boolean indicating whether the repository supports
+    consistent snapshots. This field is optional for backward compatibility with
+    old metadata that may exist without it. New implementations SHOULD included
```

Comment:
Where do we discuss what we should do if it doesn't exist?

Reply:
The spec leaves this open. From [chapter 7.1](https://theupdateframework.github.io/specification/latest/#writing-consistent-snapshots):
```
Finally, the root metadata should write the Boolean "consistent_snapshot" 
attribute at the root level of its keys of attributes. 
If consistent snapshots are not written by the repository, 
then the attribute may either be left unspecified or be set to the False value. 
Otherwise, it must be set to the True value.
```